code/modules/unit_tests/spawn_humans.dm

```diff
@@ -0,0 +1,7 @@
+/datum/unit_test/spawn_humans/Run()
+    var/locs = block(run_loc_bottom_left, run_loc_top_right)
+
+    for(var/I in 1 to 5)
+        new /mob/living/carbon/human(pick(locs))
+
+    sleep(50)
```

Comment:
Why the long sleep here? Best to put a comment down

Reply:
It’s to ensure any items humans have can fully initialise, I’ll add a comment later